monsteer/plugin/spikeReport.cpp

```diff
@@ -49,19 +50,26 @@ namespace
 const uint32_t INTERNAL_TIMEOUT = 500;
 }
 
+URI toHostAndPort( const URI& uri )
+{
+    URI out;
+    out.setHost( uri.getHost( ));
+    out.setPort( uri.getPort( ));
+    return out;
+}
+
 SpikeReport::SpikeReport( const brion::SpikeReportInitData& pluginData )
-    : _uri( pluginData.getURI( ))
+    : _uri( toHostAndPort( pluginData.getURI( )))
```

Comment:
Stefan's proposal looks wrong. Since the scheme is used to register and identify the plugin, the zeq::URI constructor will copy the "monsteer://" from pluginData.getURI(). I don't know if that's really a problem, but I think  that's what Raphael is trying to avoid here.

What I meant is:
Remove

    : _uri( toHostAndPort( pluginData.getURI( )))

And add to the contructor body this:

    _uri.setHost( uri.getHost( ));
    _uri.setPort( uri.getPort( ));